src/main/java/com/asap/server/service/MeetingService.java

```diff
@@ -82,4 +85,30 @@ public void confirmMeeting(
         meeting.setStartTime(meetingConfirmRequestDto.getStartTime());
         meeting.setEndTime(meetingConfirmRequestDto.getEndTime());
     }
+
+    @Transactional(readOnly = true)
+    public MeetingScheduleResponseDto getMeetingSchedule(Long meetingId){
+        Meeting meeting = meetingRepository.findById(meetingId)
+                .orElseThrow(() -> new NotFoundException(Error.MEETING_NOT_FOUND_EXCEPTION));
+        List<AvailableDateResponseDto> availableDateResponseDtoList = meeting.getDateAvailabilities()
+                .stream()
+                .map(DateAvailability -> new AvailableDateResponseDto(
+                        DateAvailability.getMonth(),
+                        DateAvailability.getDay(),
+                        DateAvailability.getDayOfWeek()))
```

Comment:
```suggestion
                .map(dateAvailability -> new AvailableDateResponseDto(
                        DateAvailability.getMonth(),
                        DateAvailability.getDay(),
                        DateAvailability.getDayOfWeek()))
```

p3: 이거 변수처럼 사용하는 듯 싶은데, 소문자는 어떨까요?